### ISSUE TITLE
Having issue with proper Typescript setup, help needed.

### ISSUE THREAD

**Issue by mblasee** [ctrlplusb/easy-peasy#644]
Hey All,
I am developing a react native application and using easy-peasy for state management. I really love this lib, excellent work!
However I have issues setting up the Typescript interfaces properly for multiple stores. I've created a codesandbox example of my issue.
https://codesandbox.io/s/reverent-lichterman-hef2i

I am having a multi store setup(StoreModelA.tsx, StoreModelB.tsx) and I am joining them later(Store.tsx).
I am keeping the Typescript interface declarations in the StoreModelA/B and the interface for joining the store models in Store.tsx. 
My problem is that in StoreModelA/B looks like I am missing something because my interfaces are not applies there and linter throws error when trying to access state variables or accessing actions. Also linter shows error in Store.tsx when I try to assign interface declarations to my store variables.

tl;dr having typescript issues in Store.tsx, StoreModelA.tsx and StoreModelB.tsx, please help me I'm desperate.

**Comment by ctrlplusb** [comment]
Hi @mblasee 

You were very close, you just needed to assign your store interfaces to the variables.

https://codesandbox.io/s/peaceful-wind-cdqcq?file=/src/strore/Store.tsx

```diff
- const StoreModelA = {
+ const StoreModelA: IStoreModelA = { 
   todosA: [],
   testA: false,
```

**Comment by mblasee** [comment]
Hi @ctrlplusb,
This did the trick, thank you very much for the help!
